src/styles/partials/components/_logo-type.scss

```diff
@@ -0,0 +1,31 @@
+.logotype{
+    border-right: 1px solid $gray-light;
+
+    a {
+        text-decoration: none;
+    }
+  
+    h1 {
+        padding-left: 2.2rem;
```

Comment:
please use `$padding-jumbo` here for now. @sdurphy can complain in the pr if we need to go bigger.

Reply:
done....sorry Sam